src/library_wasmfs.js

```diff
@@ -5,6 +5,18 @@
  */
 
 mergeInto(LibraryManager.library, {
+  $MEMFS: {
+    createBackend: (opts) => (_wasmfs_create_memory_backend())
+  },
+  $ICASE: {
+    createBackend: (opts) => {
```

Comment:
These days we can use the JS method syntax here:

```
createBackend(opts) {
   ...
}
```

Saves a little on code size.